ansible/roles/debops.grafana/defaults/main.yml

```diff
@@ -0,0 +1,193 @@
+# configuration file.
+grafana__default_configuration:
+
+  - name: 'server'
+    options:
+      - http_port: '{{ grafana__port }}'
+        domain: '{{ grafana__fqdn }}'
+        # TODO find a way to manage this
+        #root_url: '{{ "http://%(domain)s" }} '
+
+                                                                   # ]]]
+# .. envvar:: grafana__configuration [[[
+#
+# List of configuration options defined on all hosts in the Grafana inventory.
+grafana__configuration: []
+
+                                                                   # ]]]
+# .. envvar:: grafana__combined_configuration [[[
+#
+# Actual list of Grafana configuration options passed to the
+# configuration template. This list defines the order in which the options from
+# different variables are processed.
+grafana__combined_configuration: '{{ grafana__default_configuration + grafana__configuration }}'
+
+                                                                   # ]]]
+                                                                   # ]]]
+# Configuration for other Ansible roles [[[
+# -----------------------------------------
+
+# .. envvar:: grafana__ferm__dependent_rules [[[
+#
+# Configuration for :ref:`debops.ferm` Ansible role.
+grafana__ferm__dependent_rules:
+
+  - type: 'accept'
+    dport: [ 'grafana' ]
+    saddr: '{{ grafana__allow }}'
+    accept_any: False
+    weight: '40'
+    role: 'grafana'
+    rule_state: '{{ "absent" if (grafana__nginx_enabled | bool) else "present" }}'
+
+                                                                   # ]]]
+# .. envvar:: grafana__etc_services__dependent_list [[[
+#
+# Configuration for the :ref:`debops.etc_services` Ansible role.
+grafana__etc_services__dependent_list:
+
+  - name: 'grafana'
+    port: '{{ grafana__port }}'
+
+                                                                   # ]]]
+# .. envvar:: grafana__keyring__dependent_apt_keys [[[
+#
+# Configuration for the :ref:`debops.keyring` Ansible role.
+grafana__keyring__dependent_apt_keys:
+
+  - id: '{{ grafana__upstream_apt_id }}'
+    repo: '{{ grafana__upstream_apt_repository }}'
+    state: '{{ "present" if (grafana__deploy_type == "upstream") else "absent" }}'
+
+                                                                   # ]]]
+# .. envvar:: grafana__nginx__dependent_upstreams [[[
+#
+# Upstream configuration for the :ref:`debops.nginx` Ansible role.
+grafana__nginx__dependent_upstreams:
+  - name: 'grafana'
+    server: [ 'localhost' ]
+    port: '{{ grafana__port }}'
+
+                                                                   # ]]]
+# .. envvar:: grafana__nginx__dependent_servers [[[
+#
+# Server configuration for the :ref:`debops.nginx` Ansible role.
+grafana__nginx__dependent_servers:
+  - name: ['{{ grafana__fqdn }}']
+    by_role: 'debops.grafana'
+    filename: 'debops.grafana'
+    location:
+      '/': |
+        proxy_pass http://grafana;
```

Comment:
Try using the `proxy` server type instead, with `proxy_pass` parameter that points to the `grafana` upstream.

Reply:
used 
type: proxy
proxy_pass: 'http://grafana/'